core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -177,7 +177,7 @@ <h2 class="oppia-dashboard-title" translate="I18N_DASHBOARD_CREATOR_DASHBOARD"><
                   <[exploration.title || DEFAULT_EMPTY_TITLE]>
                 </a>
               </td>
-              <td colspan="4"
+              <td colspan="3"
```

Comment:
This change is from one of your previous commits. Please sync your develop branch with Oppia's develop branch and always create a new branch for separate PRs. Here's a [guide to sync your branch](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)

Reply:
I am having trouble in removing this commit

Reply:
Hi @suicide11, please just change the value back to 4 and then make a new commit. Everything will then be fine. Thanks!